App.js

```diff
@@ -35,6 +35,14 @@ class App extends React.Component {
 			case "wizard":
 			default:
 				let config = cloneDeep( Config );
+				let endpoint = {
```

Comment:
The naming seems a bit off.. an endpoint doesn't have headers and (correct me if i'm wrong) an endpoint _is represented_ by a url.